neurokit2/signal/signal_psd.py

```diff
@@ -125,3 +109,199 @@ def signal_psd(
         return ax
     else:
         return data
+
+
+# =============================================================================
+# Multitaper method
+# =============================================================================
+def _signal_psd_multitaper(
+    signal, sampling_rate=1000, min_frequency=0, max_frequency=np.inf
+):
+    try:
+        import mne
+
+        power, frequency = mne.time_frequency.psd_array_multitaper(signal,
+            sfreq=sampling_rate,
+            fmin=min_frequency,
+            fmax=max_frequency,
+            adaptive=True,
+            normalization="full",
+            verbose=False,
+        )
+    except ImportError:
+        raise ImportError(
+            "NeuroKit warning: signal_psd(): the 'mne'",
+            "module is required for the 'mne' method to run.",
+            "Please install it first (`pip install mne`).",
+        )
+    return power, frequency
```

Comment:
note that in all of the methods you return frequency, power and here the other way around. not sure if intentional ^^

Reply:
yeahh, i changed this to standardize with others =))